src/index.ts

```diff
@@ -154,15 +138,15 @@ export class Client {
     }
 
     // Helper Functions
-    public getRequest (path: string) {
+    public async getRequest (path: string) {
         return this.request(path);
     }
 
 //
 // HAVE A NAP
 //
 
-    private delay(ms: number) {
+    private async delay(ms: number) {
```

Comment:
since you don't use any `await` in this you don't need `async` before the Name

but I'd still suggest the method Name to be `delayAsync`  that - then its known that it can / should be awaited